tests/data/hypersphere_data.py

```diff
@@ -351,6 +419,45 @@ def log_after_exp_smoke_test_data(self):
     def log_after_exp_test_data(self):
         return super().log_after_exp_test_data(amplitude=gs.pi / 2.0)
 
+    def ricci_tensor_shape_test_data(self):
+        return self._ricci_tensor_shape_test_data(
+            self.metric_args_list, self.space_list
+        )
+
+    def scalar_curvature_shape_test_data(self):
+        return self._scalar_curvature_shape_test_data(
+            self.metric_args_list, self.space_list
+        )
+
+    def covariant_riemann_tensor_is_skew_symmetric_1_test_data(self):
+        return self._covariant_riemann_tensor_is_skew_symmetric_1_test_data(
+            self.metric_args_list, self.space_list, self.n_points_list
+        )
+
+    def covariant_riemann_tensor_is_skew_symmetric_2_test_data(self):
+        return self._covariant_riemann_tensor_is_skew_symmetric_2_test_data(
+            self.metric_args_list, self.space_list, self.n_points_list
+        )
+
+    def covariant_riemann_tensor_bianchi_identity_test_data(self):
+        return self._covariant_riemann_tensor_bianchi_identity_test_data(
+            self.metric_args_list, self.space_list, self.n_points_list
+        )
+
+    def covariant_riemann_tensor_is_interchange_symmetric_test_data(self):
+        return self._covariant_riemann_tensor_is_interchange_symmetric_test_data(
+            self.metric_args_list, self.space_list, self.n_points_list
+        )
+
+    def sectional_curvature_shape_test_data(self):
+        return self._sectional_curvature_shape_test_data(
+            self.metric_args_list,
+            self.n_points_list,
+            self.space_list,
+            self.shape_list,
+            self.n_tangent_vecs_list,
+        )
+
```

Comment:
This is old style. We should remove this private methods that create data.

Reply:
done.